schemas/3.0.0.json

```
@@ -2594,6 +2594,10 @@
                     "type": "string",
                     "description": "A brief summary of the message."
                 },
+                "version": {
+                    "type": "string",
+                    "description": "A version for the message."
```

Slight suggestion, feel free to ignore:

```suggestion
                    "description": "Version of the message."
```

---

same as above, I will apply it to the definitions. Thanks for the review